DEVELOPER_GUIDE.md

```diff
@@ -0,0 +1,25 @@
+# Developer Guide
+
+## Getting started
+
+```
+1. npm install
+
+2. npm run bootstrap
+```
+
+### Angular
+
+- `npm run start:angular-demo`
+- See http://localhost:4200
+
```

Comment:
@wkeizer You should be able to pull the branch and follow these instructions ;-) Could you try and give it a go?

This npm start command will start watch mode on two new packages: **atmention-angular** (library) and **atmention-angular-demo** (app for debugging purposes)

Reply:
Works, I have a Hello World :)